code/modules/hydroponics/gene_modder.dm

```diff
@@ -424,10 +429,14 @@
 
 /obj/item/disk/plantgene/Initialize()
 	. = ..()
-	add_overlay("datadisk_gene")
```

Comment:
add an update_icon() call here in case it starts with a gene inside for a map or something